wallet/README.md

```diff
@@ -133,6 +133,24 @@ Both sources agree that the coin exists, is worth 100, and is owned by Shawn.
 
 Let's "split" this coin by creating a transaction that spends it and creates two new coins worth 40 and 50, burning the remaining 10.
 
+### Minting coins
+It is possible to mint the coins via wallet.
+We can optionally pass the amount and public key of the owner as arguments to mint_coins.
+If optional arguments are not passed below are the default values:
+the amount is 100 coin
+owner(public key) is 0xd2bf4b844dfefd6772a8843e669f943408966a977e3ae2af1dd78e0f55f4df67 i.e SHAWN_PUB_KEY
+
+```sh
+$ tuxedo-template-wallet mint-coins \
+ --owner f41a866782d45a4d2d8a623a097c62aee6955a9e580985e3910ba49eded9e06b \
+ --amount 700 \
+
+[2024-01-16T07:23:31Z INFO  tuxedo_template_wallet] Number of blocks in the db: 66
+[2024-01-16T07:23:32Z INFO  tuxedo_template_wallet] Wallet database synchronized with node to height 260
+Minted coin from storage = (Coin(700), Sr25519Signature(Sr25519Signature { owner_pubkey: 0xd2bf4b844dfefd6772a8843e669f943408966a977e3ae2af1dd78e0f55f4df67 }))
+
+Now check all the coins available in the wallet, newly minted coins must exist by the below cmd.
+
```

Comment:
Where is the end of the code block? Shouldn't there be another three ticks somewhere?

Reply:
Sorry I missed it earlier, now I added it.

Reply:
But it still isn't quite right. You have English sentences inside the code block now. You can see it rendered here https://github.com/NadigerAmit/Tuxedo-mint-coin-feature/tree/main/wallet#minting-coins

Reply:
I fixed it.
Also fixed the mint-coin section in a better place so that the user learning flow is not broken.